### DISCUSSION TITLE
Using Aergolite in Go

### DISCUSSION THREAD

**Discussion by tutiplain** [aergoio/aergolite#5] [General]
Hi, I'm new to this project and look forward to using it. I have successfully compiled Aergolite in Linux Ubuntu. However, I can find no examples on how to use the library from Go-lang. Do I just use the regular Go bindings for sqlite, or do I need to use some other bindings?  Any help is much appreciated.

**Comment by kroggen** [comment]
Check this wiki page:

[AergoLite﹢Go](https://github.com/aergoio/aergolite/wiki/AergoLite%EF%B9%A2Go)

**Reply by tutiplain** [reply]
How did I miss this? I spent two days hacking mattn/go-sqlite to get it to work with Aergolite! I did it, but it involved vendoring my go modules and changing stuff inside them, so I was not really comfortable with it. This seems a lot simpler. Thanks!
